### PR TITLE
Add troubleshooting page with PCI Bus ID issue description

### PR DESCRIPTION
**Pull request by pentschev**

### CODE REVIEW

**Review by madsbk — APPROVED** [review]

**Comment by pentschev** [comment]
Will wait until EOD for any comments and then merge it. Thanks @madsbk for the approval!

File: docs/source/troubleshooting.rst
Comment (mmccarty) [inline docs/source/troubleshooting.rst:9]: Can you line wrap this paragraph?

**Review by mmccarty — APPROVED** [review]

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
@gpucibot merge

**Comment by jakirkham** [comment]
Thanks Peter for the PR & detective work (thank Mike for helping with this)! Also thanks Mads and Mike for the reviews! 😄

**Comment by pentschev** [comment]
Thanks to everyone mentioned by @jakirkham already, and especially thanks to @ayushdg for pointing to the actual solution with `CUDA_DEVICE_ORDER=PCI_BUS_ID`!

**Pull request merged**
